src/app/components/Toolbar/Features/Account/ManageableAccount.tsx

```diff
@@ -0,0 +1,83 @@
+import { Box } from 'grommet/es6/components/Box'
+import { Button } from 'grommet/es6/components/Button'
+import { Tab } from 'grommet/es6/components/Tab'
+import { Text } from 'grommet/es6/components/Text'
+import { ResponsiveContext } from 'grommet/es6/contexts/ResponsiveContext'
+import { useContext, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import { Account } from './Account'
+import { Wallet } from '../../../../state/wallet/types'
+import { ResponsiveLayer } from '../../../ResponsiveLayer'
+import { Tabs } from 'grommet/es6/components/Tabs'
+import { DerivationFormatter } from './DerivationFormatter'
+import { uintToBase64, hex2uint } from '../../../../lib/helpers'
+import { AddressBox } from '../../../AddressBox'
+
+export const ManageableAccount = ({
+  wallet,
+  isActive,
+  onClick,
+}: {
+  wallet: Wallet
+  isActive: boolean
+  onClick: (address: string) => void
+}) => {
+  const { t } = useTranslation()
+  const [layerVisibility, setLayerVisibility] = useState(false)
+  const isMobile = useContext(ResponsiveContext) === 'small'
+  return (
+    <>
+      <Account
+        address={wallet.address}
+        balance={wallet.balance}
+        onClick={onClick}
+        isActive={isActive}
+        path={wallet.path}
+        displayManageButton={{
+          onClickManage: () => setLayerVisibility(true),
+        }}
+      />
+      {layerVisibility && (
```

Comment:
Yes; i'm not sure it's a problem after positioning it "top" instead of center https://github.com/oasisprotocol/oasis-wallet-web/blob/8eeebd546b51c124a08f1f5973057aacd00dab5d/src/app/components/Toolbar/Features/SettingsButton/index.tsx#L52-L54